app/repositories/ListRepository.scala

```diff
@@ -70,11 +70,11 @@ class ListRepository @Inject() (listCollection: ListCollection)(implicit ec: Exe
         .map {
           res =>
             if (res.writeErrors.nonEmpty && (res.n > 0 || res.nModified > 0))
-              PartialWriteFailure(res.writeErrors.map(_.index).map(index => list(index)))
+              PartialWriteFailure(list.head.listName, res.writeErrors.map(_.index))
```

Comment:
Does this method always receive a seq with one item?

Reply:
Yes we should always be expecting atleast one on update